jetty-core/jetty-xml/src/test/java/org/eclipse/jetty/xml/XmlConfigurationTest.java

```diff
@@ -2038,4 +2039,44 @@ private interface ThrowableAction
     {
         void run() throws Exception;
     }
+
+    @Test
+    public void testExecutableComparator() throws Throwable
+    {
+        xx(null);
+        yy(null);
+        zz(null);
+
+        List<Method> methods = Arrays.stream(XmlConfigurationTest.class.getMethods())
+            .filter(m -> m.getName().length() == 2)
+            .toList();
+
+        // The implementor must also ensure that the relation is transitive: ((compare(x, y)>0) && (compare(y, z)>0)) implies compare(x, z)>0
+        assertThat(EXECUTABLE_COMPARATOR.compare(methods.get(0), methods.get(1)), is(EXECUTABLE_COMPARATOR.compare(methods.get(1), methods.get(2))));
+        assertThat(EXECUTABLE_COMPARATOR.compare(methods.get(0), methods.get(1)), is(EXECUTABLE_COMPARATOR.compare(methods.get(0), methods.get(2))));
+    }
+
+    public void xx(XXX ignored)
+    {
+    }
+
+    public void yy(YYY ignored)
+    {
+    }
+
+    public void zz(ZZZ ignored)
+    {
+    }
+
+    public interface XXX
```

Comment:
I would avoid "XXX", can we use instead either `AAA`, `BBB`, etc. or `Intf`, `Base`, `Derived`?
For the same reasons we had to change EWYK into Adaptive...